src/file_handler.rs

```diff
@@ -39,8 +39,9 @@ pub trait FileManagement {
 }
 
 impl ConfigManagement for FileHandler {
-    fn config_dir_create(&self) -> io::Result<()> {
-        fs::create_dir_all(config_dir_path())
+    fn config_dir_create(&self) -> io::Result<String> {
+        fs::create_dir_all(config_dir_path()).expect("Cannot create directory");
```

Comment:
This implementation is not actually making it easier to test, but it might be useful anyway. I'd say it's _as easy_ to test since we're asserting on the value of a `io::Result()` which can be either `Ok()` or `Error()`. Changing to `Ok(String)` won't make any difference in terms of testing. But let's keep it like this :)